cypress/integration/pages/storyPage/tests.js

```diff
@@ -43,6 +45,71 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
         }
       });
     });
+
+    it('should render podcast promo if in json and should navigate to correct podcast page', () => {
+      cy.log(service);
+      if (Cypress.env('APP_ENV') !== 'local') {
+        cy.getToggles(service);
+        cy.url().then(url => {
+          const urlForData = url.includes('amp') ? url.slice(0, -4) : url;
```

Comment:
`replace` is usually a more readable way to change strings, including to remove stuff from strings by replacing it with nothing, eg:
```suggestion
          const urlForData = url.replace('.amp', '');
```

Reply:
Thanks! This change has been made.